mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -206,21 +267,33 @@ export class ExperimentListView extends Component {
             aria-label='search experiments'
             value={searchInput}
             onChange={this.handleSearchInputChange}
+            onPressEnter={this.handleLoadMore}
             data-test-id='search-experiment-input'
+            style={{
+              width: '85%',
+            }}
```

Comment:
- let's avoid explicit `style` props - we can use emotion's `css` props, best placed in `classNames` object at the bottom
- instead of "magic" percentage values, can we use `flex` + proportions or `grid` with fractions?

Reply:
I've got most of the inline style and css issues resolve. Having some trouble determining how to get the correct width for the experiment list so that the link with elipsis `...` at the end if it flows over. Pushed what seems to work ok.

Reply:
Side note on the css, I'm not very good at it. So y'all probably want to give that a thorough vetting.